.changeset/witty-buttons-admire.md

```diff
@@ -0,0 +1,7 @@
+---
+'example-backend': patch
+---
+
+Change base node image from node:18-bullseye-slim to node:18-bookworm-slim due to Docker build error on bullseye.
+
+You can apply these change to your own `Dockerfile` by replacing `node:18-bullseye-slim` with `node:18-bookworm-slim`
```

Comment:
You don't need a changeset for the `example-backend` and I think if you delete this it will get some of the build passing